services/storage/storage.go

```diff
@@ -40,6 +40,10 @@ type VirtualHardDiskProperties struct {
 	CloudInitDataSource common.CloudInitDataSource `json:"cloudInitDataSource,omitempty"`
 	// Container name
 	ContainerName *string `json:"containername,omitempty"`
+	// Source path for the VHD. Only supports url.
```

Comment:
Can we keep it consistent with Azure SDK? Here's the pointer to the Disk struct in Azure SDK: https://github.com/Azure/azure-sdk-for-go/blob/745d4a5bb9c400c77d32dbd4046331f24e3cb2c9/sdk/resourcemanager/compute/armcompute/models.go#L1453

Reply:
I am not sure how else I should add these properties. I don't think azure has a concept of downloading disk from somewhere like we are planning to do here.